test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -454,7 +454,7 @@ await CliTester.Run(new RunConfiguration
                 {
                     using (var client = new HttpClient() { BaseAddress = new Uri("http://localhost:7073") })
                     {
-                        await Task.Delay(TimeSpan.FromSeconds(2));
+                        await Task.Delay(TimeSpan.FromSeconds(5));
```

Comment:
does the test fail if the delay isn't log enough?

Reply:
yes